language-design.md

```diff
@@ -38,7 +101,14 @@ How do the themes of _Growing a Language_ relate to the "sound lab" we did this
 
 **Response**
 
-
+The big theme from _Growing a Language_ is to let languages be growable
+and let users be able to contribute. In one sense, the theme is apparent in the
+sound lab because we are using Python, a very growable language. But at the same
+time, the API that was given to us was inflexible because the design
+decisions already made. As a result, the API itself was not as growable or
+accessible to the users. For example, the reverse function did not just reverse
+a sound and save it, it also played the sound and the reversed sound which was
+not necessarily what we wanted to do.
```

Comment:
I think another thing to keep in mind is that, for a DSL at least, you might (and probably should) make some conscious design decisions that inhibit growing your language in certain directions. This helps it stay focused on your desired domain and tasks. I wouldn't say this is always true (for DSLs, and for GPLs especially). What are some examples of when you would or would not want to allow extensibility?